src/org/aion/api/impl/ContractController.java

```diff
@@ -94,7 +94,7 @@ public ApiMsg createFromSource(final String source, final Address from, final lo
                             .valueOf(value) + " inputParams#" + String.valueOf(params));
         }
 
-        if (nrgLimit < 0 || nrgPrice < 1) {
+        if (nrgLimit < 0 || nrgPrice < 0) {
```

Comment:
base on the kernel nrg definition. we should change it. The nrglimit no less than 21K no greater than M
the nrg price no less than 10amp and no greater than 9 AION